yadm

```diff
@@ -461,7 +461,7 @@ EOF
     -v distro="$local_distro" \
     -v distro_family="$local_distro_family" \
     -v source="$input" \
-    -v source_dir="$(dirname "$input")" \
+    -v source_dir="$(builtin_dirname "$input")" \
```

Comment:
This was just a drive-by cleanup I noticed, from reading `builtin_dirname`'s comment it seems like it should be more portable?